kafka_topic_dumper/main.py

```diff
@@ -66,6 +66,11 @@ def parse_command_line():
                                  'will not load it again, it will only reset '
                                  'offsets for this consumer-group.')
 
+    reload_cmd.add_argument('-T', '--transformer',
```

Comment:
This option creates a problem in hot-reload feature. As you can change the transformer class for the same dump file we need to create a force reload option.